torch/_dynamo/utils.py

```diff
@@ -1013,30 +1013,29 @@ def record_compilation_metrics(
     exc_type: Optional[Type[BaseException]],
     exc_value: Optional[BaseException],
 ):
-    def us_to_s(field):
-        metric = metrics.get(field, None)
+    def us_to_s(m: Dict[str, Any], field: str) -> Optional[float]:
+        metric = m.get(field, None)
         return metric / 1e6 if metric is not None else None
 
-    def us_to_ms(field):
-        metric = metrics.get(field, None)
+    def us_to_ms(m: Dict[str, Any], field: str) -> Optional[int]:
+        metric = m.get(field, None)
         return metric // 1000 if metric is not None else None
 
-    def _convert_collection_to_str(field: str) -> Optional[str]:
+    def collection_to_str(m: Dict[str, Any], field: str) -> Optional[str]:
         def safe_str(item: Any) -> str:
             try:
                 return str(item)
             except Exception:
-                return str(None)
+                return "<unknown>"
```

Comment:
I could put these back to None if you prefer. I just thought it would be easier to spot real problems since many of our current scuba columns are often None.